### ISSUE TITLE
Arduplane 3.0.1+ port

### ISSUE THREAD

**Issue by hamishfagg** [MegaPirateNG/ardupilot-mpng#168]
Hi,

Do you plan on porting any versions of arduplane after 2.78?

Sorry, didn't know where else to ask this.

Thanks

**Comment by smurfy** [comment]
yes, there are plans for it. but time is the problem :) also it seems one of my flight controllers just broke so i had to use my plane's crius for my copter. but hopefully i will somehow fix the flight controller and can port a new version of arduplane as-well.

**Comment by hamishfagg** [comment]
In the mean time it is not possible to compile ardupilot-mpng 3.0.1 for planes correct?

My AIOP v2 has stopped connecting to MP after an update and I believe it is due to a new MAVLink version.

**Comment by smurfy** [comment]
Try ApmPlanner2

**Comment by hamishfagg** [comment]
Apmplanner doesnt work either =(

**Comment by hamishfagg** [comment]
I've just tried some older versions of mission planner and still no luck.
I've found that I can't get the terminal to connect either. Sometimes it gets to say the ram, version, and stops halfway through the 'load_all took xxx' line.
It just prints "load_al" and then nothing more.

Any ideas?

**Comment by SirAlex** [comment]
Usually, this problem caused by selecting CRIUS V2 board but you don't have it.
CRIUS v2 has Dataflash chip onboard and it's initialization procedure will hang if you have no this chip.

**Comment by hamishfagg** [comment]
I've selected Cirius V1 / Hobbyking AIOP boards in flashtool, flashed 2.76.
I'm getting somewhere now, but I still have some issues connecting.

Where's the best place to get proper support for this instead of spamming git issues?

**Comment by ghost** [comment]
it would be great if you could port as a final  the Plane Version 3.0.1
Regards MABABAAR

**Comment by stansan** [comment]
i have same isseu.
Megapirate plane 2.76 (from flashtool) won't connect to mission planner or other ground station software.
2.74 R4 from flashtool do connect. also the copter version which i used before when the board was on a y6 it was working.
Only the latest from flastool 2.76 don't connect.....
HK multiewii and megapirate AIO v2 i used btw.
